### PR TITLE
[web]Don't expose Firebase username and password to the client

### PR DESCRIPTION
**Pull request by ashishb**
### Description

I will send a follow-on PR to replace the secrets.

### Tested

Tested locally with `packages/web $ yarn run dev`

### Related issues

- Fixes #https://github.com/celo-org/celo-labs/issues/31

### CODE REVIEW

File: packages/web/server/FirebaseClient.ts [hunk 1]
```diff
@@ -31,7 +21,7 @@ async function getDB(): Promise<firebase.database.Database> {
 // Don't do this. It hangs next.js build process: https://github.com/zeit/next.js/issues/6824
 // const db = firebase.database()
 
-const NETWORK = 'alfajores'
+export const NETWORK = 'alfajores'
```
Comment (aaronmgdr) [inline packages/web/server/FirebaseClient.ts:24]: idea: should we eventually move this to the env-config so dev.celo.org can point to staging alfanet? ?

**Review by aaronmgdr — APPROVED** [review]
If you feel inclined to, move the firebaseclient file out from the server directory and into the  'src' directory.

**Comment by ashishb** [comment]
> If you feel inclined to, move the firebaseclient file out from the server directory and into the 'src' directory.

This is going to be a lot of small changes in quite a few files. Better to do it in a separate PR.

**Comment by ashishb** [comment]
> If you feel inclined to, move the firebaseclient file out from the server directory and into the 'src' directory.

Please file another issue for that. I don't think it is urgent but we can wait on it.

**Pull request merged**